torchtext/experimental/transforms.py

```diff
@@ -0,0 +1,105 @@
+import torch
+import torch.nn as nn
+from torchtext.experimental import functional as F
+from typing import List, Tuple
+
+
+__all__ = [
+    'BasicEnglishNormalize',
+    'RegexTokenizer'
+]
+
+
+class BasicEnglishNormalize(nn.Module):
+    r"""Basic normalization for a string sentence.
+
+    Normalization includes
+        - lowercasing
+        - complete some basic text normalization for English words as follows:
+            - add spaces before and after '\''
+            - remove '\"',
+            - add spaces before and after '.'
+            - replace '<br \/>'with single space
+            - add spaces before and after ','
+            - add spaces before and after '('
+            - add spaces before and after ')'
+            - add spaces before and after '!'
+            - add spaces before and after '?'
+            - replace ';' with single space
+            - replace ':' with single space
+            - replace multiple spaces with single space
+
+    Examples:
+        >>> import torch
+        >>> from torchtext.experimental.transforms import BasicEnglishNormalize
+        >>> test_sample = 'Basic English Normalization for a Line of Text'
+        >>> basic_english_normalize = BasicEnglishNormalize()
+        >>> jit_basic_english_normalize = torch.jit.script(basic_english_normalize)
+        >>> tokens = jit_basic_english_normalize.forward(test_sample)
```

Comment:
Do you need to call `.forward` function? You should be able to call `jit_regex_tokenizer(test_sample)` directly. Please check it as it works for most `nn.Module` class.

Reply:
Done

Reply:
You'll still need to update the comment

Reply:
Should be fixed now